upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -1329,6 +1329,9 @@ There you can find links to upgrade notes for other versions too.
     - these constants were removed so you might need to update your application appropriately:
         - `Roles::ROLE_ADMIN_AS_CUSTOMER`
 
+- fixed your version of jms/translation-bundle to 1.4.4 ([#1732](https://github.com/shopsys/shopsys/pull/1732))
```

Comment:
```suggestion
- fix your version of jms/translation-bundle to 1.4.4 in order to prevent problems with translations dumping ([#1732](https://github.com/shopsys/shopsys/pull/1732))
```